### DISCUSSION TITLE
skoala/intro/features/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3538] [Announcements]
# skoala/intro/features/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/skoala/intro/features/

**Comment by zhoujw1974** [comment]
支持通过 Service Mesh 的虚拟服务、目标规则、网关规则在网格中治理微服务流量。
